blobstore-basics/src/main/java/org/jclouds/examples/blobstore/basics/MainApp.java

```diff
@@ -69,35 +73,47 @@
 
    public static void main(String[] args) throws IOException {
 
-      if (args.length < PARAMETERS)
+      String provider;
+      String identity;
+      String credential;
+      String containerName;
+      String endpoint = null;
+      boolean isSwiftv1 = Boolean.FALSE;
+      
+      List<String> parameters = Lists.newArrayList(args);
+      if (parameters.size() < PARAMETERS)
          throw new IllegalArgumentException(INVALID_SYNTAX);
 
       // Args
-
-      String provider = args[0];
-
+      provider = parameters.get(0);
       // note that you can check if a provider is present ahead of time
       checkArgument(contains(allKeys, provider), "provider %s not in supported list: %s", provider, allKeys);
-
-      String identity = args[1];
-      String credential = args[2];
-      String containerName = args[3];
+      identity = parameters.get(1);
+      credential = parameters.get(2);
+      containerName = parameters.get(3);
+      if (parameters.size() > 4) endpoint = parameters.get(4);
+      if (parameters.size() > 5) isSwiftv1 = Boolean.parseBoolean(parameters.get(5));
 
       // Init
-      BlobStoreContext context = ContextBuilder.newBuilder(provider)
-                                               .credentials(identity, credential)
-                                               .buildView(BlobStoreContext.class);
+      ContextBuilder contextBuilder = ContextBuilder.newBuilder(provider)
+              .credentials(identity, credential);
+      if (isSwiftv1) {
+         Properties properties = new Properties();
+         properties.setProperty("jclouds.keystone.credential-type", "tempAuthCredentials");
```

Comment:
Can we add instructions to the README to somehow set this property when using Swift v1? Or change the 6th argument to be less Swift-specific, and more like "if you set this argument, we will add it as a property"?. Then you might invoke this as:
```
java -jar .... jclouds.keystone.credential-type=tempAuthCredentials
```
rather than:
```
java -jar .... true
```
?